omnibus/cookbooks/firezone/recipes/force_renewal.rb

```diff
@@ -17,7 +17,7 @@
 acme_home = "#{node['firezone']['var_directory']}/#{server}/#{keylength}/acme"
 
 # Remove cronjob (if cronjob doesn't exist no harm is done)
```

Comment:
Is this comment still correct?

I think I added this to prevent having an outdated cronjob, is this not needed anymore with the `force cronjob`?

Reply:
I think it's just a typo -- removed.